curriculum/challenges/english/01-responsive-web-design/css-balance-sheet/step-010.md

```diff
@@ -0,0 +1,76 @@
+---
+id: 6193e2baa04690265435e25f
+title: Step 10
+challengeType: 0
+dashedName: step-10
+---
+
+# --description--
+
+Below the final `.notes` element, create a `p` element with the `class` set to `row total`.
```

Comment:
It's a bit unfortunate how `row total` gets displayed here:
![image](https://user-images.githubusercontent.com/60067306/142415359-a811ad25-b6bb-4b67-b868-bbbc1b860c45.png)